client/task-list/tasks/payments/paypal.js

```diff
@@ -1,118 +1,159 @@
+/* global ppcp_onboarding */
 /**
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
 import apiFetch from '@wordpress/api-fetch';
-import { Button, CheckboxControl } from '@wordpress/components';
-import { Component, Fragment } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import { Component, Fragment, useEffect } from '@wordpress/element';
 import { compose } from '@wordpress/compose';
 import interpolateComponents from 'interpolate-components';
 import { withDispatch, withSelect } from '@wordpress/data';
 import { isEmail } from '@wordpress/url';
-import { Form, Link, Stepper, TextControl } from '@woocommerce/components';
+import { Form, Link, TextControl, Stepper } from '@woocommerce/components';
 import { getQuery } from '@woocommerce/navigation';
-import {
-	PLUGINS_STORE_NAME,
-	OPTIONS_STORE_NAME,
-	WC_ADMIN_NAMESPACE,
-} from '@woocommerce/data';
+import { PLUGINS_STORE_NAME, OPTIONS_STORE_NAME } from '@woocommerce/data';
 
-export class PayPal extends Component {
+const PAYPAL_PLUGIN = 'woocommerce-paypal-payments';
+const WC_PAYPAL_NAMESPACE = '/wc-paypal/v1';
+
+/**
+ * Loads the onboarding script file into the dom on the fly.
+ *
+ * @param {string} url of the onboarding js file.
+ * @param {Object} data required for the onboarding script, labeled as PayPalCommerceGatewayOnboarding
+ * @param {Function} onLoad callback for when the script is loaded.
+ */
+function loadOnboardingScript( url, data, onLoad ) {
+	try {
+		// eslint-disable-next-line camelcase
+		if ( ppcp_onboarding ) {
+			onLoad();
+		}
+	} catch ( e ) {
+		const script = document.createElement( 'script' );
+		script.src = url;
+		document.body.append( script );
+
+		// Callback after scripts have loaded.
+		script.onload = function () {
+			onLoad();
+		};
+		window.PayPalCommerceGatewayOnboarding = data;
+	}
+}
+
+function PaypalConnectBtn( { connectUrl } ) {
```

Comment:
Should be `PaypalConnectButton` (avoid short names)

Reply:
Fixed in: https://github.com/woocommerce/woocommerce-admin/pull/6261/commits/73fc01e1916268425e74163e8fc86209fb03a332